vcbuild.bat

```diff
@@ -236,61 +236,61 @@ if %target_arch%==x64 if %msvs_host_arch%==amd64 set vcvarsall_arg=amd64
 @rem also if both are x86
 if %target_arch%==x86 if %msvs_host_arch%==x86 set vcvarsall_arg=x86
 
-@rem Look for Visual Studio 2017
-:vs-set-2017
-if defined target_env if "%target_env%" NEQ "vs2017" goto vs-set-2019
-echo Looking for Visual Studio 2017
-call tools\msvs\vswhere_usability_wrapper.cmd "[15.0,16.0)"
-if "_%VCINSTALLDIR%_" == "__" goto msbuild-not-found
+@rem Look for Visual Studio 2019
+:vs-set-2019
+if defined target_env if "%target_env%" NEQ "vs2019" goto vs-set-2017
+echo Looking for Visual Studio 2019
+call tools\msvs\vswhere_usability_wrapper.cmd "[16.0,17.0)"
+if "_%VCINSTALLDIR%_" == "__" goto vs-set-2017
 if defined msi (
-  echo Looking for WiX installation for Visual Studio 2017...
+  echo Looking for WiX installation for Visual Studio 2019...
   if not exist "%WIX%\SDK\VS2017" (
```

Comment:
```suggestion
   if not exist "%WIX%\SDK\VS2019" (
```
? (Please check, I don't have VS/WIX for 2019 installed.)

Reply:
I just installed the latest version of the WiX toolset and there is no VS2019 SDK directory:
![image](https://user-images.githubusercontent.com/2352663/67116765-6d961a00-f1e1-11e9-8041-2220b9e17d1e.png)